backend/test/resolvers/userQuery.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-unused-expressions */
+
+import { expect } from 'chai'
+import db from '../../config/sequelize'
+import User from '../../models/user'
+import { user, users } from '../../resolvers/queries/userQuery'
+import { fakeUser } from '../factories'
+import { STUDENT, JUDGE, ADMIN } from '../../permissionLevels'
+
+describe('User Queries', function () {
+  beforeEach(function (done) {
+    db.sync({force: true}).then(() => {
+      User
+        .destroy({where: {}})
+        .then(() => done())
+    })
+  })
+  afterEach(function (done) {
+    User
+      .destroy({where: {}})
+      .then(() => done())
+  })
+
+  describe('User query', function () {
+    it('Allows users to find themeselves', function (done) {
+      fakeUser({username: 'user1'}).then((u) => {
+        user('', {id: u.username}, {auth: {type: 'STUDENT', id: 'user1'}})
+          .then((result) => {
+            expect(result.username).to.equal(u.username)
+            done()
+          })
+      })
+    })
+    it('Does not allow non admins to find other users by username', function (done) {
+      fakeUser({username: 'user1'}).then((u) => {
```

Comment:
What is u in this case?

Reply:
user, it's a u because if I write 'user' then the function I'm testing can't run. Do you want me to change it.